tests/test_swarm_cli.py

```
@@ -0,0 +1,105 @@
+"""
+CLI tests for swarm-run command.
+"""
+
+import json
+
+from click.testing import CliRunner
+
+from src.cli import cli
+
+
+def _extract_json_from_output(output: str):
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `_extract_json_from_output`

(ANN202)